api_tests/src/community.spec.ts

```diff
@@ -257,11 +257,10 @@ test("Admin actions in remote community are not federated to origin", async () =
 
 test("moderator view", async () => {
   // register a new user with their own community on alpha and post to it
-  let otherUser: API = {
-    auth: (await registerUser(alpha)).jwt ?? "",
-    client: alpha.client,
-  };
-  expect(otherUser.auth).not.toBe("");
+  let registerUserRes = await registerUser(alpha);
+  let otherUser = new LemmyHttp("http://127.0.0.1:8541", {
```

Comment:
I see, in that case it might be good to abstract the baseURL so that it can be reused.